tools/osx-uninstall.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# Added to help for removal of node on OSX
+# Referencing issue: https://github.com/nodejs/node/issues/7371
+# Using gist from rvagg: https://github.com/rvagg/io.js/blob/fhemberger-refactor-mac-installer-rebase/tools/osx-pkg/scripts/uninstall.sh
+
+if [ -z "$DIR_PREFIX" ]; then
+  DIR_PREFIX=/usr/local
+fi
+
+if [ -f /tmp/nodejs-run-uninstall ]; then
+  rm -f $DIR_PREFIX/bin/node
+  rm -f $DIR_PREFIX/bin/npm
+  rm -rf $DIR_PREFIX/include/node
+  rm -f $DIR_PREFIX/lib/dtrace/node.d
+  rm -f $DIR_PREFIX/share/man/man1/node.1
+  rm -f $DIR_PREFIX/share/systemtap/tapset/node.stp
+  rm -f $DIR_PREFIX/share/doc/node/gdbinit
+
+  rm -rf $DIR_PREFIX/lib/node_modules/npm
+  if [ ! "$(ls -A $DIR_PREFIX/lib/node_modules 2> /dev/null)" ]; then
+    rm -rf $DIR_PREFIX/lib/node_modules
+  fi
+
+  rm -f /tmp/nodejs-run-uninstall
```

Comment:
Removed because even if we dont it probably isnt needed. But what are your thoughts on recommending the curl vs making a node command?

The danger in the node command is triggering it by accident and also odd outcomes from tools like NVM and N that manage node for you.